src/modelfree/visualize/pretty_gym_compete.py

```diff
@@ -0,0 +1,178 @@
+import collections
+import ctypes
+import re
+
+from PIL import Image, ImageDraw, ImageFont
+import gym
+import mujoco_py
+import numpy as np
+
+from modelfree.envs import VICTIM_INDEX
+from modelfree.envs.gym_compete import env_name_to_canonical, game_outcome
+
+VICTIM_OPPONENT_COLORS = {
+    'Victim': (128, 128, 128, 255),
+    'Opponent': (255, 0, 0, 255),
```

Comment:
Thanks for the suggestion, I think I want to A/B test this with people. I agree having more orthogonal codings would increase the effective information bandwidth. However, I like that right now in the "common case" of the adversary in the opponent position and a zoo victim, then the adversary is all red and the victim is all grey. I think it might be harder for people to remember that e.g. white = victim, black = attacker.